tools/CouchDB_Import_Demographics.php

```diff
@@ -102,7 +102,7 @@ class CouchDBDemographicsImporter {
 
     function __construct() {
         $this->SQLDB = Database::singleton();
```

Comment:
add the \

Reply:
It's not in a namespace, I don't think it'll make a difference?